src/pages/learn.tsx

```diff
@@ -0,0 +1,331 @@
+import React, { FunctionComponent, useState } from "react";
+import { Helmet } from "react-helmet";
+import { ResourceWebinar } from "../components/UI/ResourceWebinar";
+import { ResourceLink } from "../components/UI/ResourceLink";
+import { URLS } from "../constants";
+import { getPaginatedPagesTotal, getPaginatedPosts, Pagination } from "@govtechsg/tradetrust-ui-components";
+
+const documentations = [
+  {
+    title: "Browse TradeTrust documentation",
+    url: URLS.DOCS,
+  },
+];
+
+const sourceCodes = [
+  {
+    title: "View TradeTrust source code",
+    url: URLS.GITHUB,
+  },
+];
+
+const webinars = [
+  {
```

Comment:
there will always be these 6 webinar items, not gonna change long term. is it still necessary to paginate? im thinking that current 5 per post is not giving justice to the last sixth item (been hidden in page 2, yet 1 item only also)

if paginate is still prefered, then maybe lets do 3 per post, reason being the first 3 are fundamentals and the last 3 are more advanced topics. wdyt?

Reply:
Done. currently we set it to 6 per page so the pagination won't show